tasks/process_flakes.py

```diff
@@ -77,7 +67,7 @@ def run_impl(
         """
         log.info(
             "Received process flakes task",
-            extra=dict(repoid=repo_id, commit=commit_id),
+            extra=dict(repoid=repo_id),
```

Comment:
Do you still want to log the possible commit_shas?

Reply:
as the `repo_id` (along with name, and owner info) is part of the log context now, I don’t think we need to add this context explicitly at all.